modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/EmailNotificationSettings.tsx

```diff
@@ -5,165 +5,110 @@
 
 import ClayForm, {ClayCheckbox} from '@clayui/form';
 import ClayIcon from '@clayui/icon';
+import ClayPanel from '@clayui/panel';
 import {ClayTooltipProvider} from '@clayui/tooltip';
 import {FormError, Input} from '@liferay/object-js-components-web';
 import {InputLocalized} from 'frontend-js-components-web';
 import React from 'react';
 
 import {NotificationTemplateError} from '../EditNotificationTemplate';
+import {PrimaryRecipient} from './PrimaryRecipients';
+import {SecondaryRecipient} from './SecondaryRecipients';
+import {Sender} from './Sender';
 
+import './EmailNotificationSettings.scss';
```

Comment:
Hey @larissacribeiro, could you add a blank line here?

Reply:
done, thanks!